### PR TITLE
Gemfile cleanup

### PR DESCRIPTION
**Pull request by edmilsonefs**
This is the result file for the execution of the gem **Eefgilm**.

Related to: #1369 .
Fixes #1404 .

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,118 +1,111 @@
...
 group :test do
-  gem 'capybara' # Simulates user actions for cucumber
-  gem 'cucumber-rails', :require => false # Cucmber features
-  gem 'capybara-webkit'  # Headless driver for capybara
-  gem 'selenium-webdriver' # Headful driver for capybara
-  gem 'poltergeist'  # yet another headless driver for capybara
+  gem 'capybara'
+  gem 'capybara-webkit'
+  gem 'capybara-screenshot'
+  gem 'cucumber-rails', :require => false
```
Comment (diraulo) [inline Gemfile]: For consistency let's use the Ruby 1.9 syntax for hashes.. ie: `require: false`

File: Gemfile [hunk 2]
```diff
@@ -1,118 +1,111 @@
...
-  gem 'delorean' # mocking Time in tests, aka time travelling
-  gem 'vcr' # records and plays http interactions for testing
-  gem 'shoulda-matchers', require: false # simplifies tests of common Rails functionality
-  gem 'capybara-screenshot' # creates screenshots from capybara failures
-  gem 'puffing-billy' # sandboxes network connections from javascript browser tests
+  gem 'poltergeist'
+  gem 'puffing-billy'
+  gem 'selenium-webdriver'
+  gem 'shoulda-matchers', require: false
   gem 'stripe-ruby-mock', '~> 2.3.1', :require => 'stripe_mock'
```
Comment (diraulo) [inline Gemfile]: same here and all other occurrences.

File: Gemfile (hunk 1 above)
Comment (edmilsonefs) [inline Gemfile]: I read about that in the past and forgot, thanks for remind me. This will make our gemfile more readable and clean.

**Comment by edmilsonefs** [comment]
After a conversation with @tansaku he adviced me to separate the other tasks on new issues and leave this issue here just for gemfile cleanup by using the gem mentioned above.

**Comment by edmilsonefs** [comment]
The following issues were created:
- #1411: Gemfile - Remove unused or useless gems .
- #1412: Gemfile - Categorize as much as possible .
- #1413: Gemfile - Add comments when applicable .

**Comment by edmilsonefs** [comment]
@diraulo Changes requests were sent. 👍

**Comment by edmilsonefs** [comment]
@diraulo one more thing, since my environment is not good at the moment, I just sent the Gemfile and not the .lock so you guys can ran the tests locally before submit.

**Comment by tansaku** [comment]
@edmilsonefs please run bundle install and update your gemfile.lock and commit - thanks!

**Pull request merged**